sbt-app/src/sbt-test/tests/setup-cleanup/test

```diff
@@ -4,7 +4,7 @@ $ absent tested
 $ absent cleanup
 
 # without Setup configured, the setup file won't exist and the test will fail
--> testQuick
+-> testFull
```

Comment:
Just making sure, did you mean to convert this from a cached test to a full test?

Reply:
Yea, I made the change in #7680 recently to test `testQuick` but I think `testFull` would test the actual setup capability more correctly.